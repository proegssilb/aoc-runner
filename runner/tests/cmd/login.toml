bin.name = "cargo-aoc"
args = "login"
fs.sandbox = true