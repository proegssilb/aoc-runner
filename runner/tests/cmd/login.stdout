This command doesn't implement proper authenticaion yet. Use your browser to visit and log in to the AOC website, then copy the value of the 'session' cookie, and paste it here: 
Session cookie stored successfully.
